indico/modules/events/registration/templates/management/print_badges.html

```diff
@@ -2,24 +2,31 @@
 {% from 'message_box.html' import message_box %}
 
 
-{% if registrations != all_registrations %}
-    {% call message_box('warning') -%}
-        {% trans count=(all_registrations|length - registrations|length), total=all_registrations|length -%}
-            Tickets are currently not available for the selected registration.
-        {%- pluralize total -%}
-            Tickets are currently not available for {{ count }} out of the {{ total }} selected registrations.
-        {%- endtrans %}
-    {%- endcall %}
-{% endif %}
+{% block warning_box %}
+    {% if registrations != all_registrations %}
+        {% call message_box('warning') -%}
+            {% trans count=(all_registrations|length - registrations|length), total=all_registrations|length -%}
+                Tickets are currently not available for the selected registration.
+            {%- pluralize total -%}
+                Tickets are currently not available for {{ count }} out of the {{ total }} selected registrations.
+            {%- endtrans %}
+        {%- endcall %}
+    {% endif %}
+{% endblock %}
 
 {% if registrations %}
+    {% set available_registrations = True %}
```

Comment:
```suggestion
    {% set available_registrations = true %}
```

Reply:
also this is super weird. and likely to get refactored out. why do you need this and can you find some better way of doing this?

Reply:
We need this because we are trying to use the content of this template to set the badge print settings at the category level without performing the print, so we do not need the registrations to be present and so we set it to always be true to get the rest of the content

Reply:
maybe wrap it in a block which you then override either with or without calling `{{ super() }}`?

Reply:
okay, I'll try that.